ion-schema/src/isl/isl_constraint.rs

```diff
@@ -392,6 +461,17 @@ impl IslConstraint {
         }
     }
 
+    // helper method to convert from a string to offset in minutes
+    fn offset_minutes(s: &str, range: Range) -> IonSchemaResult<i32> {
```

Comment:
Isn't this duplicated in `util.rs`? Is it necessary to have it in both places?

Reply:
I have removed this method from both the places with the new compact solution you suggested